ta/miss/include/ta_miss_test.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2014, STMicroelectronics International N.V.
```

Comment:
- This copyright doesn't look right, I think it should be your own, although this is mostly copied from other TA of course.
- As for the licensing text, we prefer using a SPDX tag. Contrary to optee_os, we have not added tags to all the existing files in optee_test, but in my opinion we'd better use them for new files. They're just less cumbersome.

So all in all the comment block of this file should be:
```C
// SPDX-License-Identifier: BSD-2-Clause
/*
 * Copyright (c) 2019 <you or your company>
 */
```

Reply:
This was indeed a miss from my side. The new iteration has all the copyright in place.